### ISSUE TITLE
Remove the devel styling

### ISSUE THREAD

**Issue by AbrarSL** [flathub/xyz.diamondb.gtkcord4#6]
I'm not sure why the devel styling is used? Is there a build option to disable it?

**Comment by ghost** [comment]
I'll see what I can do

**Comment by ghost** [comment]
progress/update: (mostly for my own reference)
- the devel styling shows when git reports that there are changes: [gotkit/app.IsDevel](https://github.com/diamondburned/gotkit/blob/main/app/app.go#L48)
- adding `-buildvcs=false` right [here](https://github.com/flathub/xyz.diamondb.gtkcord4/blob/master/xyz.diamondb.gtkcord4.yml#L29) fully disables this, however it might break the version in the 'about' dialog, so this would only be a temporary fix. There should be a way to build the flatpak without modifying the contents of the repo (one xml file iirc).
- I'm unable to test the build with this option anyway because of this error:
```
panic: Something in this program imports go4.org/unsafe/assume-no-moving-gc to declare that it assumes a non-moving garbage collector, but your version of go4.org/unsafe/assume-no-moving-gc hasn't been updated to assert that it's safe against the go1.21 runtime. If you want to risk it, run with environment variable ASSUME_NO_MOVING_GC_UNSAFE_RISK_IT_WITH="go1.21" set. Notably, if go1.21 adds a moving garbage collector, this program is unsafe to use.

goroutine 1 [running]:
go4.org/unsafe/assume-no-moving-gc.init.0()
	/run/build/gtkcord4/vendor/go4.org/unsafe/assume-no-moving-gc/untested.go:25 +0x1da
```
I'm looking for a way to use go1.20 inside the flatpak manifest file, instead of latest go1.21, among other alternatives.

**Comment by ghost** [comment]
![image](https://github.com/flathub/xyz.diamondb.gtkcord4/assets/111794344/74222eaa-e97b-4bb9-b0e5-882d0de5fb22)
with https://github.com/diamondburned/gtkcord4/pull/149 and `-buildvcs=false` it compiles + does not show the devel CSD, however it also does not show version number.

**Comment by ghost** [comment]
should be fixed with #8
